installscripts/jazz-terraform-unix-noinstances/scripts/modifyCodebase.sh

```diff
@@ -25,7 +25,7 @@ function inArray() {
    return 0
 }
 
-lambda_services=("jazz_cognito-authorizer" "jazz_cloud-logs-streamer" "jazz_services-handler" "jazz_events-handler" "jazz_environment-event-handler" "jazz_deployments-event-handler" "jazz_asset-event-handler" "jazz_slack-event-handler""jazz_es-kinesis-log-streamer" "jazz_splunk-kinesis-log-streamer")
+lambda_services=("jazz_cognito-authorizer" "jazz_cloud-logs-streamer" "jazz_services-handler" "jazz_events-handler" "jazz_environment-event-handler" "jazz_deployments-event-handler" "jazz_asset-event-handler" "jazz_slack-event-handler""jazz_es-kinesis-log-streamer" "jazz_splunk-kinesis-log-streamer" "jazz_root-authorizer")
```

Comment:
Function/service name being used is `jazz_cognito-admin-authorizer` as per https://github.com/tmobile/jazz/pull/697. Requires an update?

Reply:
Updated the name.